packages/icons-angular/package.json

```diff
@@ -16,52 +16,52 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "@angular/common": "^15.0.1",
-    "@angular/core": "^15.0.1",
-    "@ant-design/colors": "^5.0.0"
+    "@angular/common": "^16.0.0",
+    "@angular/core": "^16.0.0",
+    "@ant-design/colors": "^7.0.0"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^15.0.1",
-    "@angular-eslint/builder": "15.1.0",
-    "@angular-eslint/eslint-plugin": "15.1.0",
-    "@angular-eslint/eslint-plugin-template": "15.1.0",
-    "@angular-eslint/schematics": "15.1.0",
-    "@angular-eslint/template-parser": "15.1.0",
-    "@angular/animations": "^15.0.1",
-    "@angular/cli": "^15.0.1",
-    "@angular/compiler": "^15.0.1",
-    "@angular/compiler-cli": "^15.0.1",
-    "@angular/forms": "^15.0.1",
-    "@angular/language-service": "^15.0.1",
-    "@angular/platform-browser": "^15.0.1",
-    "@angular/platform-browser-dynamic": "^15.0.1",
-    "@angular/router": "^15.0.1",
+    "@angular-devkit/build-angular": "^16.0.0",
+    "@angular-eslint/builder": "^16.0.0",
+    "@angular-eslint/eslint-plugin": "^6.0.0",
```

Comment:
This should be `^16.0.0`. Please make sure that CI passes.

Reply:
well spotted, I'll fix right away